pad_api_data/pad_etl/data/dungeon.py

```diff
@@ -30,88 +30,80 @@ def __init__(self, raw: List[Any]):
         self.bgm1 = raw[5]
         self.bgm2 = raw[6]
         self.rflags2 = int(raw[7])
-        # These need to be parsed depending on flags
-        self.otherModifier = raw7_map[int(raw[7])]
-
-        possibleDrops = {}
-
-        # This next loop runs through the elements from raw[8] until it hits a 0. The 0 indicates the end of the list
-        # of drops for the floor, the following segments are the dungeon modifiers
-        pos = 8
-
-        while (int(raw[pos]) is not 0):
-            rawVal = int(raw[pos])
-            if rawVal > 10000:
-                val = rawVal - 10000
-                possibleDrops[val] = "rare"
-                pos += 1
-            else:
-                possibleDrops[rawVal] = "normal"
-                pos += 1
-        pos += 1
-        modifiers = getModifiers(raw, pos)
-
-        drops = []
-        dropRarities = []
-
-        for key, val in possibleDrops.items():
-            drops.append(key)
-            dropRarities.append(val)
 
-        self.drops = drops
-        self.dropRarities = dropRarities
-
-        self.entryRequirement = modifiers.entryRequirement
-        self.requiredDungeon = modifiers.requiredDungeon
-
-        self.modifiers = modifiers.modifiers
+        # These need to be parsed depending on flags
+        self.other_modifier = raw7_map[int(raw[7])]
+
+        try:
+            modifiers = getModifiers(raw)
+        except Exception as e:
+            print("Error:", e, "on parse of values", int(raw[8:]))
```

Comment:
Does this ever happen on a parse of the current dungeon file?  If it does, then this change shouldn't be committed.

Also this is probably the wrong place to swallow this exception. I'd rather it propagate up and the caller decide what to do.